packages/ii-login-button/vite.config.js

```diff
@@ -0,0 +1,9 @@
+import { defineConfig } from 'vite';
```

Comment:
there is vite in the repo as well?

Reply:
Oh I had it for a bit for previewing the button, but I removed the dependency in deference to a docs situation